10_PredictionExamples.ipynb

```diff
@@ -0,0 +1,420 @@
+{
```

Comment:
<p>There is no <code>ys</code>  in this model, should be <code>y</code>.  You also should describe why you want to predict those -- these are the latent states of the HMM, and so it's useful to predict them, because we can make all kinds of useful guesses about our problem when we know the states.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>